service/controller/v25/ebs/ebs.go

```diff
@@ -78,50 +81,56 @@ func (e *EBS) DeleteVolume(ctx context.Context, volumeID string) error {
 // specified the instance will be stopped first.
 func (e *EBS) DetachVolume(ctx context.Context, volumeID string, attachment VolumeAttachment, force bool, shutdown bool, wait bool) error {
 	if shutdown {
-		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("stopping instance %s", attachment.InstanceID))
+		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("requesting to stop instance %#q", attachment.InstanceID))
 
 		i := &ec2.StopInstancesInput{
 			InstanceIds: []*string{
 				aws.String(attachment.InstanceID),
 			},
 		}
+
 		_, err := e.client.StopInstances(i)
 		if err != nil {
 			return microerror.Mask(err)
 		}
 
-		if wait {
-			e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waiting for instance %s being stopped", attachment.InstanceID))
+		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("requested to stop instance %#q", attachment.InstanceID))
+	}
 
-			i := &ec2.DescribeInstancesInput{
-				InstanceIds: []*string{
-					aws.String(attachment.InstanceID),
-				},
-			}
-			err := e.client.WaitUntilInstanceStopped(i)
-			if err != nil {
-				return microerror.Mask(err)
-			}
+	if shutdown && wait {
+		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waiting for instance %#q being stopped", attachment.InstanceID))
 
-			e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waited for instance %s being stopped", attachment.InstanceID))
+		i := &ec2.DescribeInstancesInput{
+			InstanceIds: []*string{
+				aws.String(attachment.InstanceID),
+			},
 		}
 
-		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("stopped instance %s", attachment.InstanceID))
+		err := e.client.WaitUntilInstanceStopped(i)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waited for instance %#q being stopped", attachment.InstanceID))
```

Comment:
```suggestion
		e.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waited for instance %#q to stop", attachment.InstanceID))
```

This shorter but still clear I reckon. WDYT?

Reply:
Yeah nice. I changed that. Makes it more consistent with the other logs above as well.